### PR TITLE
br: fix unstable integration tests

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:
1. make the test get the correct stats
2. fix unstable integration br_s3 tests. make s3 outage during write files.

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/task/backup.go [hunk 1]
```diff
@@ -678,6 +678,22 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 			}
 		}()
 	}
+
+	failpoint.Inject("s3-outage-during-writing-file", func(v failpoint.Value) {
+		log.Info("failpoint s3-outage-during-writing-file injected, " +
+			"process will sleep for 3s and notify the shell to kill s3 service.")
```
Comment (BornChanger) [inline br/pkg/task/backup.go:684]:
```suggestion
			"process will sleep for 5s and notify the shell to kill s3 service.")
```

**Review by BornChanger — APPROVED** [review]
rest lgtm

**Review by Leavrth — APPROVED** [review]

**Pull request merged**